Documentation/Blazorise.Docs/Pages/News/2023-01-15-release-notes-120.razor

```diff
@@ -197,22 +197,24 @@
 </Paragraph>
 
 <Heading Size="HeadingSize.Is3">
-    DataGrid 
+    DataGrid
 </Heading>
 
 <Paragraph>
-    A new <Code Tag>Groupable</Code> feature is introduced. You are now able to group the data in the <Code Tag>DataGrid</Code>.
+    The new grouping feature for <Link To="docs/extensions/datagrid">Blazorise DataGrid</Link> allows you to easily group and organize your data by specific columns. This can be useful for analyzing large sets of data and finding patterns or trends.
```

Comment:
might want to use these phrasing on the seo too :D

Reply:
Exactly haha